### ISSUE TITLE
Can nsqd support many SUBs on one connection?

### ISSUE THREAD

**Issue by jinq0123** [nsqio/nsq#1238]
I have many topics. Each topic subcription results one connection. I tried send 2 SUB on one connection, but failed: cannot SUB in current state. It would be nice to allow many SUBs on one connection.

**Comment by ploxiln** [comment]
Yeah, that's a feature that is not offered, and probably won't be any time soon, sorry.
